src/main/java/codeu/model/data/BotArguments.java

```diff
@@ -0,0 +1,13 @@
+package codeu.model.data;
+
+import java.util.ArrayList;
+
+/** Class representing BotArguments, the information required for a bot to work.*/
+public class BotArguments {
+  private ArrayList<Object> objectList;
```

Comment:
Should this be an `ArrayList` of `Object`s or `String`s? I generally would prefer to avoid "untyped" `Objects` that then get typecasted by implementers. It removes type safety from the compiler